pkgs/tools/admin/chkservice/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, ninja, pkg-config, systemd, ncurses, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "chkservice";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "linuxenko";
+    repo = "chkservice";
+    rev = version;
+    hash = "sha256:0dfvm62h6dwg18f17fn58nr09mfh6kylm8wy88j00fiy13l4wnb6";
+  };
+
+  # Tools needed during build time
+  nativeBuildInputs = [
+    cmake
+    # Makes the build faster, adds less than half a megabyte to the build
+    # dependencies
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    systemd
+    ncurses
+  ];
+
+  hardeningDisable = [ "format" ];
```

Comment:
Maybe it would be worth adding there is a build issue with this hardening feature enabled

Reply:
I would argue that it is exactly what this line suggests, as it is the only reason to add it. But I can add a comment.

Reply:
I think in an ideal world we'd be fixing the package upstream by opening a PR, and then it would make a lot of sense to add a comment here linking to the PR. But since nobody wants to bother with that for such old packages it doesn't matter imo.